lib/beaker-hostgenerator/hypervisor/docker.rb

```diff
@@ -17,6 +17,33 @@ def generate_node(node_info, base_config, bhg_version)
           base_config['image'] = "amd64/#{base_config['image']}"
         end
 
+        docker_commands = []
+
+        case node_info['ostype']
+        when /^ubuntu/
+          docker_commands << 'cp /bin/true /sbin/agetty'
+
+          if node_info['ostype'] =~ /14.04/
+            docker_commands << 'rm /usr/sbin/policy-rc.d'
+            docker_commands << 'rm /sbin/initctl; dpkg-divert --rename --remove /sbin/initctl'
+          end
+
+          extra_packages_to_install = case node_info['ostype']
+                                      when /1404/
+                                        []
+                                      when /1604/
+                                        ['locales']
+                                      else
+                                        ['locales', 'iproute2', 'gnupg']
+                                      end
+
+          docker_commands << "apt-get install -y net-tools wget apt-transport-https #{extra_packages_to_install.join(' ')}"
```

Comment:
apt-transport-https a transitional package starting at least 16.04: https://packages.ubuntu.com/bionic/apt-transport-https. It was still a real package in Debian 9 but in Debian 10 it's also a transitional package.

Reply:
yeah it seems to be a noop on 16.04 and newer, good catch!